docs/local-lxd/explanation/ssh-debug.md

```diff
@@ -0,0 +1,18 @@
+# SSH Debug
+
+SSH debugging allows a user to identify and resolve issues or errors that occur through the secure
+shell (SSH) connection between a client and a server.
+
+To enhance the security of the runner and the infrastructure behind the runner, only user ssh-keys
```

Comment:
Does it make more sense to say "user SSH keys" here? Unless ssh-keys is referring to a specific configuration or item?

Reply:
I would leave this fixes for future PRs, as this is not specific to the PR (and it is quite big by now)